### PR TITLE
@sweir27 => [Documentation] Add doc about rolling back

### PR DESCRIPTION
**Pull request by mzikherman**
This is a doc for rolling back, it's better than what we have now (🙈), curious if someone who hasn't rolled back before feels like they understand the steps after looking thru this.

### CODE REVIEW

File: docs/rolling_back.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Locate the corresponding Circle CI deploy process that corresponded to the deploy you are rolling back to. You may need to scroll back in the build history until you find it. One such example is: https://circleci.com/workflow-run/9fec809a-d096-4fc0-88b7-82419ada519d
+
+1. Open up the final step, `deploy_hokusai_production`, and expand the 'Echo Manifest' step. This is a config variable that changes with each deploy, and needs to be set manually during the rollback.
+
+![echo manifest](images/echo_manifest.png "Echo Manifest")
+
+1. Copy the long value into a text editor, and add single quotes around the entire thing. Then copy it to your clipboard.
+
+1. Run the following command to set the config in the environment. `hokusai production env set ASSET_MANIEST=...clipboard contents...`
```
Comment (alloy) [inline docs/rolling_back.md]:
```suggestion
1. Run the following command to set the config in the environment. `hokusai production env set ASSET_MANIFEST=...clipboard contents...`
```

File: docs/rolling_back.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Run `hokusai registry images`. You'll see output like the below.
+
+![list of images](images/hokusai_images.png "Hokusai Images")
+
+1. Locate the name of a tag you want to deploy. In this case, since we're rolling back, it'll be the previous tag of the form `production-YYYY-mm-dd..`
+
+![specific image](images/single_image.png "Last Prod Image")
+
+1. Locate the corresponding Circle CI deploy process that corresponded to the deploy you are rolling back to. You may need to scroll back in the build history until you find it. One such example is: https://circleci.com/workflow-run/9fec809a-d096-4fc0-88b7-82419ada519d
```
Comment (joeyAghion) [inline docs/rolling_back.md]: ...the ~~corresponding~~ CircleCI deploy process that corresponded...

File: docs/rolling_back.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Open up the final step, `deploy_hokusai_production`, and expand the 'Echo Manifest' step. This is a config variable that changes with each deploy, and needs to be set manually during the rollback.
+
+![echo manifest](images/echo_manifest.png "Echo Manifest")
+
+1. Copy the long value into a text editor, and add single quotes around the entire thing. Then copy it to your clipboard.
+
+1. Run the following command to set the config in the environment. `hokusai production env set ASSET_MANIEST=...clipboard contents...`
+
+1. When that completes, run the following command to deploy the desired image that you've located above. `hokusai deploy production--2018-11-21--20-05-37`
```
[the remark this answers is not in the document]
Comment (mzikherman) [reply docs/rolling_back.md]: You can actually make a 'Suggestion' on GitHub using this (see https://github.com/artsy/force/pull/3123#discussion_r236027096), which will add that as a commit here!

File: docs/rolling_back.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+## How to Rollback (on Kubernetes, deployed w/ Hokusai)
+
+So, you're reading this doc, in which case either nothing is on fire and you're simply educating yourself (yay!), or, there's a problem and you need to rollback Force. Take a deep breath, and read on.
+
+1. Run `hokusai registry images`. You'll see output like the below.
+
+![list of images](images/hokusai_images.png "Hokusai Images")
+
+1. Locate the name of a tag you want to deploy. In this case, since we're rolling back, it'll be the previous tag of the form `production-YYYY-mm-dd..`
```
Comment (sweir27) [inline docs/rolling_back.md]: Not sure a suggestion will work since it's many lines, but these are all `1.`!

File: docs/rolling_back.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Open up the final step, `deploy_hokusai_production`, and expand the 'Echo Manifest' step. This is a config variable that changes with each deploy, and needs to be set manually during the rollback.
+
+![echo manifest](images/echo_manifest.png "Echo Manifest")
+
+1. Copy the long value into a text editor, and add single quotes around the entire thing. Then copy it to your clipboard.
+
+1. Run the following command to set the config in the environment. `hokusai production env set ASSET_MANIFEST=...clipboard contents...`
+
+1. When that completes, run the following command to deploy the desired image that you've located above. `hokusai production deploy production--2018-11-21--20-05-37`
```
Comment (sweir27) [inline docs/rolling_back.md]: Ohhh this makes sense. So you set the manifest and then deploy, which refreshes the pods with the new config + rolls back to the desired image. I think we did this in reverse order last time!

**Comment by sweir27** [comment]
Small comment but other than that feel free to merge!

File: docs/rolling_back.md (hunk 5 above)
Comment (mzikherman) [reply docs/rolling_back.md]: Yep!

**Comment by mzikherman** [comment]
Merging in!

File: docs/rolling_back.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Run `hokusai registry images`. You'll see output like the below.
+
+![list of images](images/hokusai_images.png "Hokusai Images")
+
+2. Locate the name of a tag you want to deploy. In this case, since we're rolling back, it'll be the previous tag of the form `production-YYYY-mm-dd..`
+
+![specific image](images/single_image.png "Last Prod Image")
+
+3. Locate the Circle CI deploy process that corresponded to the deploy you are rolling back to. You may need to scroll back in the build history until you find it. One such example is: https://circleci.com/workflow-run/9fec809a-d096-4fc0-88b7-82419ada519d
```
Comment (oxaudo) [inline docs/rolling_back.md]: Ok... I'm the one who has never done that before. All the steps are pretty clear, this finding the `process corresponding to the deploy found in step 1` is a bit not clear. How does one know that this is a corresponding deploy? The deploy does not seem to have any tags that mention `production-YYYY-mm-dd..` or I'm just missing it. I found the one just by looking at prod deploy on that day, but that might be confusing if there are several in . close time frame.

Maybe there can be an image here that shows how does one know that this deploy corresponds to the tag found above?

Comment (mzikherman) [reply docs/rolling_back.md]: Good point @oxaudo ! That is exactly how I found the corresponding Circle CI process...I scrolled back in the history on Force until I was on the day of that last deploy...and then looked at the list for a Circle CI `release` branch build.

You're totally right that can possibly be made easier, and can be confusing with multiple deploys on that day, etc.

File: docs/rolling_back.md (hunk 4 above)
Comment (alloy) [reply docs/rolling_back.md]: I recently learned this; you can leave them all `1.` and markdown auto-numbers them, making it trivial to move them around.

**Pull request merged**